plugins/flytekit-dbt/README.md

```diff
@@ -0,0 +1,15 @@
+# Flytekit dbt plugin
+
+Flytekit plugin for performing DBT tasks. Currently it supports both `dbt run` and `dbt test` tasks.
+
+To install the plugin, run the following command:
+
+```bash
+pip install flytekitplugins-dbt
+```
+
+_Example coming soon!_
```

Comment:
how long do you think this will be here for?  over under on a year?

Reply:
Good question. Do you think it's worth pointing this to a blog post or the flytesnacks examples? (when those are created obviously).